### PR TITLE
SS-3810 Short keys if only one phrase removed auto populating

### PR DESCRIPTION
**Pull request by sheikrxnt**

### CODE REVIEW

File: src/helpers/shortkey/suggestion/index.js [hunk 1]
```diff
@@ -264,18 +264,33 @@ function getSuggestionComponent() {
...
+        trigger,
+        startingCharacter,
+        endingCharacter,
+        placeholderKeyPairs
+      } = config;
       const selectedMention = this.filteredSuggestions[activeOption];
       if (selectedMention) {
-        console.log('firing addMention');
-        addMention(editorState, onChange, separator, trigger, selectedMention,startingCharacter ,endingCharacter ,placeholderKeyPairs );
+        console.log("firing addMention");
```
Comment (sneu012) [inline src/helpers/shortkey/suggestion/index.js:277]: Remove

File: src/helpers/shortkey/suggestion/index.js [hunk 2]
```diff
@@ -264,18 +264,33 @@ function getSuggestionComponent() {
...
+          placeholderKeyPairs
+        );
       }
     };
-    
-    getSanatizedHtml = (html) => {
-      return ;
 
-    }
+    getSanatizedHtml = html => {
```
Comment (sneu012) [inline src/helpers/shortkey/suggestion/index.js:291]: Remove unused function

File: src/helpers/shortkey/suggestion/index.js [hunk 3]
```diff
@@ -87,10 +86,14 @@ class Suggestion {
...
               this.config.trigger = currentTrigger;
+
+              if (mentionText && mentionText.trim().length <= 0) {
+                return false;
+              }
               return (
                 suggestion.phrase
                   .toLowerCase()
-                  .indexOf(mentionText && mentionText.toLowerCase()) >= 0
+                  .indexOf(mentionText.toLowerCase()) >= 0
```
Comment (sneu012) [inline src/helpers/shortkey/suggestion/index.js:96]: Do we need to check if mentionText exists before we convert it to lowercase?

File: src/helpers/shortkey/suggestion/index.js (hunk 1 above)
Comment (sheikrxnt) [reply src/helpers/shortkey/suggestion/index.js]: Removed

File: src/helpers/shortkey/suggestion/index.js (hunk 2 above)
Comment (sheikrxnt) [reply src/helpers/shortkey/suggestion/index.js]: Removed

File: src/helpers/shortkey/suggestion/index.js (hunk 3 above)
Comment (sheikrxnt) [reply src/helpers/shortkey/suggestion/index.js]: For the safer side now added the mentionText exists before we convert it to lowercase

**Review by sneu012 — APPROVED** [review]

**Pull request merged**